pkg/install/users.go

```diff
@@ -73,22 +73,11 @@ func DeleteControllerUsers(clusterConfig *v1beta1.ClusterConfig) error {
 // EnsureUser checks if a user exists, and creates it, if it doesn't
 // TODO: we should also consider modifying the user, if the user exists, but with wrong settings
 func EnsureUser(name string, homeDir string) error {
-	exists, err := users.CheckIfUserExists(name)
-	if err != nil {
-		return err
-	}
-
-	if exists {
-		return nil
+	_, err := users.GetUID(name)
+	if _, ok := err.(user.UnknownUserError); ok {
```

Comment:
I missed this originally -- this could be `errors.Is()` as well. Not holding this up though.

Reply:
doh... I think we should fix this too.